src/client/components/OrganizationExportMenu.tsx

```diff
@@ -0,0 +1,56 @@
+/** @jsx jsx */
+import { jsx, Box } from "theme-ui";
+import { Button as MenuButton, Wrapper, Menu, MenuItem } from "react-aria-menubutton";
+import Icon from "../components/Icon";
+import { OrganizationSlug } from "../../shared/entities";
+import { style } from "./MenuButton.styles";
+import store from "../store";
+import { exportProjects, exportOrgUsers } from "../actions/organization";
+
+enum UserMenuKeys {
+  ExportMaps = "maps",
+  ExportUsers = "users"
+}
+
+interface Props {
+  readonly slug: OrganizationSlug;
+}
+
+const OrganizationExportMenu = (props: Props) => {
+  return (
+    <Wrapper
+      sx={{ position: "relative", display: "inline-block", ml: 3, pr: 1 }}
+      onSelection={(userMenuKey: string) => {
+        const action = userMenuKey === UserMenuKeys.ExportMaps ? exportProjects : exportOrgUsers;
+        store.dispatch(action(props.slug));
+      }}
+    >
+      <MenuButton
+        sx={{
+          ...{ variant: "buttons.secondary", fontWeight: "light" },
+          ...style.menuButton
+        }}
+        className="export-menu"
+      >
+        <Icon name="export" />
+        Download
+      </MenuButton>
+      <Menu sx={style.menu}>
+        <ul sx={style.menuList}>
+          <li key={UserMenuKeys.ExportMaps}>
+            <MenuItem value={UserMenuKeys.ExportMaps}>
+              <Box sx={style.menuListItem}>Download Maps</Box>
+            </MenuItem>
+          </li>
+          <li key={UserMenuKeys.ExportUsers}>
+            <MenuItem value={UserMenuKeys.ExportUsers}>
+              <Box sx={style.menuListItem}>Download Users</Box>
+            </MenuItem>
+          </li>
```

Comment:
I recently got burned by copy-pasting markup for another project (repeated an id instead of changing it in a spot where it was copied) and so I would favor this:
```suggestion
          {[
            { key: UserMenuKeys.ExportMaps, label: "Download Maps" },
            { key: UserMenuKeys.ExportUsers, label: "Download Users" }
          ].map(({ key, label }: { readonly key: UserMenuKeys; readonly label: string }) => (
            <li key={key}>
              <MenuItem value={key}>
                <Box sx={style.menuListItem}>{label}</Box>
              </MenuItem>
            </li>
          ))}
```